### PR TITLE
Add here aggregation steps 

### PR DESCRIPTION
**Pull request by chmnata**

### CODE REVIEW

File: here/here_aggregation.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+- [Input Parameters](#input-parameters)  
+    - [Geometry](#Geometry)
+    - [Time Range](#time-range)
+    - [Date Range](#date-range)
+- [Output Parameters](#output-parameters)  
+- [Aggregation](#aggregation)
+    - [Using congestion tables](#using-congetion-tables)
+    - [Using raw speed table](#using-raw-speed-table)
+
+## What is HERE data?
```
Comment (radumas) [inline here/here_aggregation.md:12]: What is HERE data should be merged into the `here/readme.md` and this aggregation readme should be referenced from it.

File: here/here_aggregation.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
+# Table of Contents  
+- [What is HERE data?](#what-is-here-data-?)  
+- [Input Parameters](#input-parameters)  
+    - [Geometry](#Geometry)
+    - [Time Range](#time-range)
+    - [Date Range](#date-range)
+- [Output Parameters](#output-parameters)  
+- [Aggregation](#aggregation)
+    - [Using congestion tables](#using-congetion-tables)
```
Comment (radumas) [inline here/here_aggregation.md:9]: Typo in the link

File: here/here_aggregation.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+- Geometry
+    - The spatial extent of the study area
+- Time range
+    - Time periods definitions, e.g. am peak, pm peak
+- Date range
+    - period of time to aggregate over
+    - Day of week, weekday or weekend
+    - Including or excluding holidays
+
+## Geometry
```
Comment (radumas) [inline here/here_aggregation.md:39]: These headings should be H3

**Review by radumas — CHANGES_REQUESTED** [review]
Great work putting this together. Some minor formatting fixes.

Did you fix the bug in the aggregation sql snippets I highlighted during your talk?

File: here/here_aggregation.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+
+```sql
+SELECT      corridor_id,
+            analysis_period,
+            time_period,
+            week,
+            sum(tti * tt_baseline) / sum(tt_baseline) AS tti,
+            sum(tti * tt_baseline) / sum(tt_baseline) * tt_baseline AS tt
+FROM        segment_tt 
+GROUP BY    corridor_id, analysis_period, time_period, week, tt_baseline
```
Comment (radumas) [inline here/here_aggregation.md:154]: I think trying to sum `tt_baseline` while grouping by it doesn't make sense.

**Review by radumas — CHANGES_REQUESTED** [review]
Some smol further changes required

File: here/here_aggregation.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+
+	FROM 		data_requests.input_table -- input table
+	GROUP BY 	input_table.uid 
+```
+
+**Step 2**: Aggregate link level travel time from 5 minutes to an hour
+
+```sql
+-- Aggregate link travel times up to an hour 
+	SELECT 		input_table.uid,
```
Comment (radumas) [inline here/here_aggregation.md:181]: In the step above this is `corridor_id`

File: here/here_aggregation.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+			    date_part('isodow'::text, a.tx)::integer <@ time_ranges.dow
+
+	GROUP BY 	input_table.uid, input_table.link_dir, datetime_bin, input_table.length, period
+```
+
+**Step 3**: Aggregate link level hourly travel time up to corridor level, where at least 80% of the corridor (by distance) has observations
+
+```sql
+-- Aggregate link level hourly travel time to corridor level
+	SELECT 		b.uid,
```
Comment (radumas) [inline here/here_aggregation.md:204]: there's no table here aliased as `b`

File: here/here_aggregation.md (hunk 5 above)
Comment (radumas) [reply here/here_aggregation.md:181]: `uid` then gets used in the rest of the steps

**Review by radumas — CHANGES_REQUESTED** [review]
sorry, some final nitpicking.

**Comment by chmnata** [comment]
re-re-re-re-review requested @radumas :meow_salute:

**Review by radumas — APPROVED** [review]
grrrrrrrreat 🐯

**Pull request merged**